chemclipse/plugins/org.eclipse.chemclipse.ux.extension.ui/src/org/eclipse/chemclipse/ux/extension/ui/swt/TaskTile.java

```diff
@@ -263,7 +263,7 @@ private void updateStyle(int style) {
 			fontSize = 18;
 		}
 		//
-		Font font = new Font(getDisplay(), "Arial", fontSize, SWT.BOLD);
+		Font font = new Font(getDisplay(), "Arial", fontSize, SWT.BOLD); //$NON-NLS-1$
```

Comment:
To harmonize using fonts, you could  use
```

org.eclipse.chemclipse.swt.ui.support.Fonts
getCachedFont(...)
```

or the default Font selection

```
org.eclipse.swtchart/src/org/eclipse/swtchart/Resources.java
Resources.DEFAULT_FONT_NAME
```

[https://github.com/eclipse/swtchart/issues/371](https://github.com/eclipse/swtchart/issues/371)

Reply:
That adds an SWTChart dependency on an unrelated part of the UI. I came up with https://github.com/eclipse/chemclipse/pull/1960 instead.